### ISSUE TITLE
More city furniture objects?

### ISSUE THREAD

**Issue by Matthias84** [StrandedKitty/streets-gl#31]
While not part of the essential public infrastructure, some mappers love micromapping, which might give the srendering interesting details:

# Advertsing
* displays
* posterbox
* billboards

# Furnitures
* benches
* bicycle parking
* shelters

# Barriers
* Fences
* Bollards

# Waste
* containers
* trash baskets

# Playgrounds
* dedicated devices

You might check out https://wiki.openstreetmap.org/wiki/3D_Model_Repository if you want to partner with them and use existing / contribute modells?

**Comment by StrandedKitty** [comment]
Thank you for your suggestions.

Fences are already rendered. Regarding other objects: most of them are too tiny and you can't really see them unless you zoom in, so for this reason I haven't implemented them. The smallest object on the map right now is a fire hydrant, and often it's incredibly hard to spot it on a sidewalk because of it's small size.

**Comment by ivanbranco** [comment]
I would like to suggest highway=street_lamp
It could also support values such as light:lit=17:30-05:30 ecc. for aumented realism since Streets GL supports dynamic lighting, but maybe is asking too much 👀

**Comment by Matthias84** [comment]
Yeah it's true that some are pretty small and might be hard to see with all surrounding other objects. On the other hand, there might be pretty unpopulated areas (like seaside, ...) where this small goodies might give some nice street orientation? http://maps.osm2world.org/?zoom=18&lat=54.07229&lon=12.11656&layers=B0FTFF
Or how about limiting to lines of objects like bollards?

**Comment by dreua** [comment]
Foutains: Coming from f4 I miss them ;)

**Comment by mnalis** [comment]
> Playgrounds:  dedicated devices

I'd too especially like some of the more popular [playground equipment](https://wiki.openstreetmap.org/wiki/Key:playground) as [playgrounds](https://www.openstreetmap.org/way/261386677) look [soo sad](https://streets.gl/#45.80623,15.92280,24.99,51.00,63.59) with only the benches, picnic tables and fences around. 

> Regarding other objects: most of them are too tiny and you can't really see them unless you zoom in, so for this reason I haven't implemented them. The smallest object on the map right now is a fire hydrant, and often it's incredibly hard to spot it on a sidewalk because of it's small size.

And they are big enough; certainly bigger than fire hydrants.

**Comment by Matthias84** [comment]
I just appended a few more features (sorry, but I couldn't resist :wink: ).
